protos/sources/vega/commands/v1/commands.proto

```diff
@@ -118,236 +106,249 @@ message IcebergOpts {
+  // New pegged offset for the order.
+  // This field is an unsigned integer scaled to the market's decimal places.
   string pegged_offset = 7;
-  // Amend the pegged order reference for the order.
+  // New pegged reference for the order.
   vega.PeggedReference pegged_reference = 8;
 }
 
-// A liquidity provision submitted for a given market
+// A command that indicates to the network the party's intention to supply liquidity to the given market and become a liquidity provider.
+// An active liquidity provider for a market will earn fees based on the trades that occur in the market.
 message LiquidityProvisionSubmission {
   reserved 4, 5;
-  // Market ID for the order.
+  // Market that the submitter wishes to provide liquidity for.
   string market_id = 1;
-  // Specified as a unitless number that represents the amount of settlement asset of the market.
+  // Amount that the submitter will commit as liquidity to the market, specified as a unitless number in the settlement asset of the market.
   // This field is an unsigned integer scaled using the asset's decimal places.
   string commitment_amount = 2;
   // Nominated liquidity fee factor, which is an input to the calculation of taker fees on the market, as per setting fees and rewarding liquidity providers.
   string fee = 3;
-  // Reference to be added to every order created out of this liquidity provision submission.
+  // Arbitrary reference to be added to every order created out of this liquidity provision submission.
   string reference = 6;
 }
 
-// Cancel a liquidity provision request
+// A command allows a liquidity provider to inform the network that they will stop providing liquidity for a market.
 message LiquidityProvisionCancellation {
-  // Unique ID for the market with the liquidity provision to be cancelled.
+  // Market that the submitter will stop providing liquidity for.
   string market_id = 1;
 }
 
-// Amend a liquidity provision request
+// A command that allows a liquidity provider to update the details of their existing liquidity commitment.
+// Any field that is left unset or as a default value indicates that this field on the original submission will be left unchanged.
 message LiquidityProvisionAmendment {
-  // Unique ID for the market with the liquidity provision to be amended.
+  // Market that the submitter provides liquidity for.
   string market_id = 1;
-
-  // From here at least one of the following is required to consider the command valid.
-  string commitment_amount = 2; // empty strings means no change
-  string fee = 3; // empty strings means no change
-  repeated vega.LiquidityOrder sells = 4; // empty slice means no change
-  repeated vega.LiquidityOrder buys = 5; // empty slice means no change
-  string reference = 6; // empty string means no change
+  // New commitment amount.
+  string commitment_amount = 2;
+  // New nominated liquidity fee factor.
+  string fee = 3;
+  repeated vega.LiquidityOrder sells = 4;
```

Comment:
@jeremyletang I'm guessing these LP order fields should really be removed like in `LiquidityProvisionSubmission`?

Reply:
ah, correct, I thought I had done this. Would you mind doing it as part of this one? and reserve them?